src/services/transfer/__tests__/transfer-out-fragments.test.js

```diff
@@ -228,7 +229,8 @@ describe('transferOutFragments', () => {
 
   it('should update fragment status to DOWNLOAD_FAILED when unable to retrieve fragment from presigned url', async () => {
     // given
-    const error = new DownloadError('error');
+    const acknowledgementErrorCode = AcknowledgementErrorCode.ERROR_CODE_10_A;
+    const error = new DownloadError(null, acknowledgementErrorCode);
```

Comment:
Does a `null` parameter not cause a null pointer?

Reply:
Same as above, this null parameter is only used in the constructor along with a null check, so it never causes problems.